sys/debug_irq_disable/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2022 Benjamin Valentin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_DEBUG_IRQ_DISABLE
```

Comment:
We're currently in the Kconfig transition phase - thus please add `depends on TEST_KCONFIG`.

To test this PR out, call `make info-modules` with and without env `TEST_KCONFIG=1` set.